radish34/ops/build_api.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
this is definitely a great improvement! `pushd` doesnt work the way it was invoked in `package.json` on ubuntu, unless it is explicitly specified in `.npmrc` - this solves it better